### PR TITLE
Edit UG to include findMember and findEvent commands

### PR DESCRIPTION
**Pull request by SelwynAng**
Edited User Guide to include `findMember` and `findEvent` command descriptions and summaries.

### CODE REVIEW

**Review by wr1159 — CHANGES_REQUESTED** [review]
I believe this is confused with viewMember and viewEvent

File: docs/UserGuide.md [hunk 1]
```diff
@@ -172,6 +172,32 @@ Examples:
...
+Finds and lists event(s) which name(s) contain the provided `KEYWORD`.
+
+Format: `findEvent KEYWORD [MORE_KEYWORDS]`
+* Finds and lists event(s) which name(s) contain the specified `KEYWORD`.
+* More than 1 `KEYWORD` can be provided to find more events.
+* The `KEYWORD` must match minimally one of the words in the name of the event to be found, where capitalisation does not matter.
+
+* Examples:
+* `findEvent party` displays the event(s) which name(s) contain 'party'
+* `findMember party marathon gaming` displays the event(s) which name(s) contain 'party', 'marathon' and/or 'gaming'.
```
Comment (andrechuakj) [inline docs/UserGuide.md]:
```suggestion
* `findEvent party marathon gaming` displays the event(s) which name(s) contain 'party', 'marathon' and/or 'gaming'.
```

**Review by andrechuakj — CHANGES_REQUESTED** [review]
Just a small correction

File: docs/UserGuide.md (hunk 1 above)
Comment (SelwynAng) [reply docs/UserGuide.md]: Ok will correct this!

**Review by wr1159 — APPROVED** [review]
LGTM

**Review by andrechuakj — APPROVED** [review]
LGTM

**Pull request merged**